pulp_container/app/serializers.py

```diff
@@ -372,6 +372,10 @@ def validate(self, data):
             manifest = models.Manifest.objects.get(
                 pk__in=new_data["latest_version"].content.all(), digest=new_data["digest"]
             )
+            # TODO Evaluate, whether we need touch here.
+            # The manifest is in a repository version, so not orphaned,
+            # but can the version be deleted between this and using it?
+            manifest.touch()
```

Comment:
Please discuss

Reply:
there is a small window for such case, so i probably would keep it.